### PR TITLE
Update analytics, add ability to run local server with the specific path

### PR DESCRIPTION
**Pull request by xmile1**
- update analytics

### CODE REVIEW

**Comment by xmile1** [comment]
@z586 I believe v6 will not use this repo anymore from this time on.

File: in-repo.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 owner:
   active:
     team: black
-    since: 2017-07-25
\ No newline at end of file
+    since: 2017-12-15
```
Comment (notaurologist) [inline in-repo.yaml]: This shouldn't change.

File: src/util/analytics.js [hunk 2]
```diff
@@ -1,6 +1,57 @@
...
+    if (
+      window.measure &&
+      window.measure.collect &&
+      typeof window.measure.collect === 'function'
+    ) {
+      window.measure.collect(name, props)
+    }
+  },
+  isUDFShareUrl (shareUrl) {
+    return shareUrl.match(/^https:\/\/(([a-z0-9-]+(\.[a-z0-9-]+)*?\.)?(invisionapp\.com|invision\.works)\/prototype)\/[A-Za-z-0-9]*(\/(preview|play|comment|inspect))?/i)
```
Comment (notaurologist) [inline src/util/analytics.js]: I'd at least put this in `src/util/url.js` for now. Would be better to do something more long-term.

File: src/util/analytics.js [hunk 3]
```diff
@@ -1,6 +1,57 @@
-export function track () {
-  if (window.analytics && window.analytics.track) {
-    window.analytics.track.apply(window.analytics, arguments)
+import config from './config'
+// When the app is initializing, window.analytics doesn't exist yet, so we can't check whether it
+// exists now. Instead, we check it on-demand.
+const serverType = config.serverType
```
Comment (z586) [inline src/util/analytics.js]: Why do we need it here and in all events props?

File: src/util/analytics.js (hunk 2 above)
Comment (z586) [reply src/util/analytics.js]: As we discussed with @xmile1 , he **intends** to get this data from API response with metadata.

File: src/util/analytics.js (hunk 3 above)
Comment (xmile1) [reply src/util/analytics.js]: Its part of the props required as per https://docs.google.com/spreadsheets/d/1iAigsPDUtWiz0O3xSPBof_OwkVY8cJy82dc54PGnF7M/edit#gid=1982570671 @z586

File: src/util/analytics.js (hunk 2 above)
Comment (xmile1) [reply src/util/analytics.js]: Yeah, I missed removing this during cleanup, I already refactored the code, I will remove this now.

File: src/util/analytics.js (hunk 3 above)
Comment (notaurologist) [reply src/util/analytics.js]: @xmile1 If required, it should at least use `ENVIRONMENT`. I don't think that needs to hold up anything, though.

**Comment by notaurologist** [comment]
@xmile1 In the future, please don't squash your commits until after review. It makes it difficult to see what was changed after feedback.

**Comment by xmile1** [comment]
ok, @notaurologist , Noted!

**Pull request merged**